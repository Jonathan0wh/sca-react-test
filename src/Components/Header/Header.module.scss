@import "styles/colors";

.header {
  background-color: $middlegrey;
  height: 50px;
  display: flex;
  justify-content: center;
  align-items: center;
}

.text {
  font-weight: bold;
  color: $lightgrey;
}
