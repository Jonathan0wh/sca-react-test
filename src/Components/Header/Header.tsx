import React from 'react';

import styles from './Header.module.scss';

const Header = () => (
  <header className={styles.header}>
    <div className="container">
      <div className="row">
        <p className={styles.text}>Subscribe</p>
      </div>
    </div>
  </header>
);

export default Header;
