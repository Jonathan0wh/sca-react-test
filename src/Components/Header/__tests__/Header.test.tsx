import React from 'react';
import { shallow } from 'enzyme';

import Header from '../Header';

it('render Header without crashing', () => {
  const wrapper = shallow(<Header />);

  expect(wrapper).toMatchSnapshot();
});
