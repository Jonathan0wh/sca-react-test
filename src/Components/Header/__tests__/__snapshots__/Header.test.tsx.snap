// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Header without crashing 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <p
        className="text"
      >
        Subscribe
      </p>
    </div>
  </div>
</header>
`;
